app/models/record.rb

```diff
@@ -12,7 +12,7 @@ class Record < ActiveRecord::Base
   validates :disposition_date, date: true, presence: true
 
   def eligible?
-    if self.misdemeanor? && (self.disposition_date > (DateTime.now - 5.years))
+    if self.misdemeanor? && (self.disposition_date > (DateTime.now - 5.years)) 
```

Comment:
Redundant `self` detected.<br>Trailing whitespace detected.<br>Do not use `DateTime.now` without zone. Use one of `Time.zone.now`, `DateTime.now.current`, `DateTime.now.in_time_zone`, `DateTime.now.utc`, `DateTime.now.getlocal`, `DateTime.now.iso8601`, `DateTime.now.jisx0301`, `DateTime.now.rfc3339`, `DateTime.now.to_i`, `DateTime.now.to_f` instead.